src/transaction.rs

```diff
@@ -467,6 +469,8 @@ pub struct Event {
 pub struct EventContent {
     pub keys: Vec<EventKey>,
     pub data: EventData,
+    /// The order of the event within the transaction
+    pub order: u64,
```

Comment:
I think the correct design is to create a struct 
```rust
pub struct OrderedEvent {
   content: EventContent,
   order: u64
}

Reply:
I made the update, sorry for the misunderstood